_posts/2020-11-17-dropping-puppet-5-and-RHEL-6-support.md

```diff
@@ -0,0 +1,13 @@
+---
+layout: post
+title: Dropping Puppet 5 and RHEL 6 support
+date: 2020-03-29
+github_username: bastelfreak
+twitter_username: BastelsBlog
+---
+
+Hi Vox Pupuli followers!
+Puppet 5 will be end of life in the next days, Puppet 7 is around the corner.
+Also RHEL/CentOS will be end of life in the next few days. In the next few days
```

Comment:
All RHEL/CentOS will be end of life?

But agree on the support dropping. If people still haven't upgraded form RHEL6 or Puppet5, they won't update their modules either, so they won't be affected by this.

Reply:
> If people still haven't upgraded form RHEL6 or Puppet5, they won't update their modules either

From experience in multiple companies over the last 5 years, I can guarantee that's exactly what people do.  As @igalic said, "upgrading a module is infinitely easier than upgrading a configuration management system"